src/Mapper.cpp

```diff
@@ -1365,8 +1365,8 @@ void Mapper::generateJSON4IncrementalMap(CGRA* t_cgra, DFG* t_dfg){
 
   jsonFile<<"     \"Tile2Level\":{"<<endl;      
   // Generates level informations of current mapping results
-  map<int, vector<CGRANode*>> FanIO_CGRANodes;
-  vector<int> FanIOs;
+  vector<int> FanIOs; // FanIO is the number of links of current CGRANode connected to other CGRANode.
```

Comment:
Plz just move the comments above the code. You can insert a line between two comments if you want.

Reply:
Same to the others.

Reply:
Has fixed all similar issues